ONSdigital/eq-author
# Digeq 7 review questionnaire

**What**
This pull request delivers the functionality for DIGEQ-7 - Review questionnaire. It will allow an user to specify whether a questionnaire has been reviewed.

**Pre-requisites**
1. A user that can access the digital eq system
2. A user which has previously added a survey and created at least one questionnaire

**How to test**
1 Check out this branch: git checkout digeq-7-review-questionnaire
2 Run the database migrations python manage.py migrate
3 Run the server: python manage.py runserver
4 Browse to http://127.0.0.1:8000/login/ and login with valid credentials
5 Browse to http://127.0.0.1:8000/surveys/
6. Select questionnaire
7. Click review, questionnaire should now display with a reviewed checked box
8. Click add question, add new question
9. Verify that questionnaire is no longer in reviewed state and the 'review' link is available again

**Who can review**
Anyone on Digital EQ apart from @warren-methods and @weapdiv-david
